src/AfriCC/EPP/AbstractFrame.php

```diff
@@ -220,7 +228,7 @@ protected function realxpath($path)
             array_unshift($path_parts, $this->mapping . ':' . $this->command);
         }
 
-        if (!empty($this->command)) {
+        if (!empty($this->command) && !$this->ignore_command) {
```

Comment:
This was required in order to succesfuly build NASKs `Report` suite if you're wondering ;)